### PR TITLE
Moved some more unit tests to Codeception

### PR DESCRIPTION
**Pull request by SidRoberts**

### CODE REVIEW

File: tests/unit/Mvc/CollectionTest.php [hunk 1]
```diff
@@ -345,4 +346,109 @@ protected function clearCollection()
...
+
+                expect($songs)->count(2);
+            }
+        );
+    }
+
+    public function testCollectionsEvents()
+    {
+        $this->specify(
+            "...",
```
Comment (sergeyklay) [inline tests/unit/Mvc/CollectionTest.php]: ? :)

File: tests/unit/Mvc/ViewTest.php [hunk 2]
```diff
@@ -817,4 +819,175 @@ protected function _getViewDisabled($level = null)
 
         return $view;
     }
+
+    public function testCacheDI()
+    {
+        $this->specify(
+            "...",
```
Comment (sergeyklay) [inline tests/unit/Mvc/ViewTest.php]: ???

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Review by sergeyklay — APPROVED** [review]

**Comment by SidRoberts** [comment]
I'll finish this off tomorrow.

**Comment by Izopi4a** [comment]
thank you mr. Roberts 👍

**Comment by SidRoberts** [comment]
I've had to add aliases to the Query Builder tests. I assume it's because the models are now in namespaces.

**Comment by SidRoberts** [comment]
This is ready to merge.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**